transport/cert/testdata/context_aware_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+	"cert_provider_command":["cat", "testdata/testcert.pem"]
+}
```

Comment:
nit: a few of these files need newlines

Reply:
Added EOF newlines to everything under testdata.